modules/document_repository/php/files.class.inc

```diff
@@ -389,7 +389,6 @@ class Files extends \NDB_Page
             'document'    => $filename,
         ];
         $uploadNotifier->notify($msg_data);
-        print_r($response);
```

Comment:
thanks

Reply:
no problem! I'm the one that left it there lol

Reply:
haha oh I thought I added it. 🤷